Makefile

```diff
@@ -46,6 +46,7 @@ prepare-ios: prepare ##@prepare Install iOS specific dependencies
 release: release-android release-ios ##@build build release for Android and iOS
 
 release-android: prod-build-android ##@build build release for Android
+	cd android; ./gradlew react-native-android:installArchives
```

Comment:
Does this need to be done each time? How about using the setup script?

Reply:
yeah had a same question here

Reply:
@jeluard i'm not familiar with makefile's syntax/structure and have no idea where that setup script lives, besides i'm not using `make`. So if you know the way to make it better please add commit to this PR

Reply:
I am not familiar with make either but probably you don't want to install this frequently.
Setup script is here https://github.com/status-im/status-react/blob/develop/scripts/setup
Please update it accordingly